src/components/reportSummary/reportSummary.styles.ts

```diff
@@ -5,6 +5,12 @@ export const styles = StyleSheet.create({
   reportSummary: {
     height: '100%',
   },
+  cardHeader: {
+    paddingBottom: '0px !important',
+  },
+  cardBody: {
+    paddingTop: '0px !important',
```

Comment:
Are we overriding the padding of PF cards? If so, perhaps we should confirm with Natalie we really want to do this?

Also, is there a way to do this without `!important` (e.g., perhaps by adding some specificity)? There are some examples of how to override PF styles in files like `awsDetails.styles.ts`.

Reply:
@dlabrecq You're right that wasnt the best approach. Instead of that, I've taken off margin under the botttom of the paragraph tag. Thats as much as I can do without overriding padding of cards

Reply:
<img width="913" alt="screen shot 2018-11-07 at 1 57 25 pm" src="https://user-images.githubusercontent.com/20118816/48153952-73774180-e295-11e8-93e0-bf59fda75c5a.png">